MTDome/v2/_init.yaml

```diff
@@ -0,0 +1,4 @@
+host: "127.0.0.1"
+port: 5000
+connection_timeout: 10
+read_timeout: 10
```

Comment:
Is the host really localhost? If not, surely you need an `_summit.yaml` file with the correct host, in addition to this new `_init.yaml` file?

Reply:
At some point we will need that, yes. Currently the MTDome EIE network has not been connected to the Rubin Observatory network so it is unclear what the hostname will be. As soon as this is known, we will add the _summit.yaml file.